api_app/script_analyzers/observable_analyzers/phishtank.py

```diff
@@ -0,0 +1,38 @@
+import requests
+import logging
+from api_app.exceptions import AnalyzerRunException
+from api_app.script_analyzers.classes import ObservableAnalyzer
+from intel_owl import secrets
+import base64
+
+logger = logging.getLogger(__name__)
+
+
+class Phishtank(ObservableAnalyzer):
+    def set_config(self, additional_config_params):
+        self.api_key_name = additional_config_params.get(
+            "api_key_name", "PHISHTANK_API_KEY"
+        )
+
+    def run(self):
+        result = {}
+        headers = {}
+        data = {
+            "url": base64.b64encode(self.observable_name.encode("utf-8")),
+            "format": "json",
+        }
+        api_key = secrets.get_secret(self.api_key_name)
+        if not api_key:
+            logger.warning(f"{self.__repr__()} -> Continuing w/o API key..")
+        else:
+            data["app_key"] = api_key
+            headers = {"User-Agent": "phishtank/IntelOwl"}
```

Comment:
shoudln't we send this header with or w/o api key, in both cases ? any particular reason why is this been sent only when there's an API key ?

Reply:
everything else, looks good. Can you please just make this little change and then I'd merge.

Reply:
That makes sense. Only issue is, the particular username IntelOwl needs to be created if we want to pass the 'User-Agent' field. I will make the change and push it immediately, but there might be errors since the username does not exist.